core/src/main/java/brooklyn/entity/rebind/dto/MementosGenerators.java

```diff
@@ -127,13 +143,16 @@ public static BrooklynMemento newBrooklynMemento(ManagementContext managementCon
     
     /**
      * Inspects an entity to create a corresponding memento.
+     * <p>
+     * @deprecated since 0.7.0, see {@link #newBasicMemento(BrooklynObject)}
      */
+    @Deprecated
```

Comment:
I've been burned by that assumption before.  I'm almost certain you're right but I think not worth it now.  We will have fun as soon as 070 is out getting rid of all these!!!